.github/workflows/master.yml

```diff
@@ -170,6 +170,54 @@ jobs:
             **/kyuubi-jdbc-engine.log*
             **/kyuubi-hive-sql-engine.log*
 
+  spark-connector-cross-version-test:
+    name: Kyuubi Spark Connector Cross Version Test
+    runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false
+      max-parallel: 6
```

Comment:
please limit it to 1, we got asked to 

> All workflows SHOULD have a job concurrency level less than or equal to 15.

https://infra.apache.org/github-actions-policy.html

Reply:
Good to know. Updated.